rclcpp/include/rclcpp/client.hpp

```diff
@@ -251,6 +257,25 @@ class ClientBase
   rclcpp::QoS
   get_response_subscription_actual_qos() const;
 
+  using IntraProcessManagerWeakPtr =
+    std::weak_ptr<rclcpp::experimental::IntraProcessManager>;
+
+  /// Implementation detail.
+  RCLCPP_PUBLIC
+  void
+  setup_intra_process(
```

Comment:
this should probably be protected

Reply:
Done in [0d17a8e](https://github.com/ros2/rclcpp/pull/1847/commits/0d17a8e2774f4121b6a2bd58ce0d34d6f6d1b987)